src/commons/application/ApplicationTypes.ts

```diff
@@ -230,16 +241,56 @@ const sourceSubLanguages: Language[] = [
 ];
 
 export const sourceLanguages: SALanguage[] = sourceSubLanguages.map(sublang => {
+  const { chapter, variant } = sublang;
+  const supportedFeatures: LanguageFeatures = sublang['supports'] ?? {};
```

Comment:
Is there a need to retrieve the `supports` object if it will always be empty and all of its fields will always be overridden?

Reply:
My bad, this was originally done as a guard against someone potentially adding a `supports` object in `sourceSubLanguages`, before I explicitly typed the array. This is not needed anymore and can just be initialized as an empty object.

Reply:
Resolved in bbe80f4ce064a2d4afb5170655c1f0fe332859bf.